src/options.ts

```diff
@@ -63,6 +64,11 @@ export interface IProps<ListItem = any> {
    */
   sectionListProps?: Animated.AnimatedProps<SectionListProps<ListItem>>;
 
+  /**
+   * A reference to a view which provides scroll behaviour
+   */
+  scrollViewRef?: React.RefObject<ScrollView | FlatList | SectionList>
```

Comment:
Could you move this to be above the callbacks section, the same order as the props from src/index.tsx

Also, could you add this props to the `docs/props.md` with the same order logic